arch/arm/core/cortex_m/mpu/arm_core_mpu.c

```diff
@@ -10,7 +10,11 @@
 #include <soc.h>
 #include <arch/arm/cortex_m/cmsis.h>
 #include <arch/arm/cortex_m/mpu/arm_core_mpu.h>
-#include <logging/sys_log.h>
+
+#define LOG_MODULE_NAME arm_core_mpu
+#define SYS_LOG_LEVEL SYS_LOG_LEVEL_DEBUG
```

Comment:
this entry has no impact on the logger configuration. If you want to overwrite default configuration you need to use
`#define LOG_LEVEL LOG_LEVEL_DEBUG`
Ideally, kconfig option should be added (see the comment https://github.com/zephyrproject-rtos/zephyr/pull/8816#discussion_r208098277)